tests/test_integrated_channels/test_utils.py

```diff
@@ -309,3 +309,25 @@ def test_truncate_item_collections(self):
         assert len(out_a) == 3
         assert len(out_b) == 5
         assert len(out_c) == 2
+
+        in_a = []
```

Comment:
i wanted to add some test cases where the first couple buckets are small or empty to make sure we'd still get the delete content (bucket c)